homeassistant/components/danfoss_air/__init__.py

```diff
@@ -0,0 +1,87 @@
+"""
+Support for Danfoss Air HRV.
+
+Configuration:
```

Comment:
I believe per the style guidelines, we no longer put configuration in the component either.

https://developers.home-assistant.io/docs/en/development_guidelines.html

Reply:
I am pretty sure I actually read somewhere it was OK to add documentation example in the header. I am however not able to find it now. Anyway I have removed the example.